packages/opentelemetry-core/src/correlation-context/propagation/HttpCorrelationContext.ts

```diff
@@ -91,13 +91,15 @@ export class HttpCorrelationContext implements HttpTextPropagator {
       return context;
     }
     const pairs = headerValue.split(ITEMS_SEPARATOR);
-    if (pairs.length == 1) return context;
     pairs.forEach(entry => {
       const keyPair = this._parsePairKeyValue(entry);
       if (keyPair) {
         correlationContext[keyPair.key] = { value: keyPair.value };
       }
     });
+    if (Object.entries(correlationContext).length <= 0) {
```

Comment:
Nit: `===0` should be enough, it can't be lower than 0

Reply:
Right, I'll change it.

Reply:
Done